opcua/server/discovery_service.py

```diff
@@ -0,0 +1,49 @@
+
```

Comment:
The name Service is a bit confusing since OPCUA services are the ones defined in spec. Mayve this is better called DiscoveryUtils DiscoveryRegistrator... I let you think about it

Reply:
We should also call services the classes offering the functions defined by spec

Reply:
OK now I read more of your code and maybe name service is correct